checker/src/visitors.rs

```diff
@@ -726,6 +726,19 @@ impl<'a, 'b: 'a, 'tcx: 'b, E> MirVisitor<'a, 'b, 'tcx, E> {
                 }
                 return;
             }
+            let result: AbstractValue = self.try_to_inline_standard_ops_func(fun, &actual_args);
```

Comment:
This code block feels super specific to get such a prominent place in visit_call(). Can you reasonable abstract it out? Mostly I'm just concerned that things like this made it hard to read visit_call() and understand generally what it is doing.

Reply:
I have not found a better way of doing it.